project/src/PreloadedScripts/SteeringSave.gd

```diff
@@ -4,6 +4,8 @@ const CONFIG_FILE = "user://input.cfg"
 const DEFAULT_CONFIG_FILE = "user://default_input.cfg"
 const INPUT_ACTIONS = ["rotation_left", "rotation_right", "rotation_up", "4directions_left", "4directions_up", "4directions_right", "4directions_down", "8directions_up", "8directions_up_left", "8directions_up_right", "8directions_left", "8directions_right", "8directions_down", "8directions_down_left", "8directions_down_right", "shot_pad", "shot_keyboard"]
 
+const GAME_VERSION = 1.3
```

Comment:
Podobnie nie game version tylko config version. Ma to też więcej sensu, że osobne pliki konfiguracyjne mają różne wersje